server/services/forms/schemas/add-standard-line.js

```diff
@@ -14,6 +15,9 @@ module.exports = options => Form()
         Component('date', 'expiry_date')
             .withValidator('required', 'Expiration date is required')
             .withValidator('isValidDate', 'Expiration date must be a valid date')
+            .withValidator('isValidDay', 'Date must contain a real day')
+            .withValidator('isValidMonth', 'Date must contain a real month')
+            .withValidator('isYearWithinRange', 'Date must be before ' + (new Date().getFullYear() + YEAR_RANGE))
```

Comment:
Will the user get this error if they enter 1782?

Reply:
No the validators are applied in order so they will get the next error "Expiration date must not be in the past" and only get the year validation error if they put something too far in the future